apps/mitt-konto/package.json

```diff
@@ -13,7 +13,6 @@
   },
   "scripts": {
     "start": "parcel index.html -p 8000",
-    "build": "yarn build-purs && parcel build index.html --no-cache && cp xd_receiver.html ./dist",
-    "build-purs": "cd ../.. && spago bundle-module -m MittKonto.Main -t apps/mitt-konto/index.js --path \"packages/*/src/**/*.purs\" --path \"apps/*/src/**/*.purs\" "
+    "build": "parcel build index.html --experimental-scope-hoisting --no-cache && cp xd_receiver.html ./dist"
```

Comment:
We'd still need a purs-building step here right? Adding a `spago build` should be a no-op if you already built, and will update the generated js if you didn't

Reply:
Yeah, maybe better have it here